shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -146,7 +149,13 @@ export default {
     providerConfig: {
       type:    Object,
       default: () => null
-    }
+    },
+
+    s3EndpointHasError: {
```

Comment:
this should be a `data` property (it's internal), rather than a component property (nothing will pass this in to the component)